src/main/java/io/github/moulberry/notenoughupdates/listener/RenderListener.java

```diff
@@ -1026,11 +1026,20 @@ public void onGuiScreenKeyboard(GuiScreenEvent.KeyboardInputEvent.Pre event) {
 		}
 		if (NotEnoughUpdates.INSTANCE.config.hidden.dev && Keyboard.isKeyDown(Keyboard.KEY_B) &&
 			Minecraft.getMinecraft().currentScreen instanceof GuiChest &&
-			((((ContainerChest) ((GuiChest) Minecraft.getMinecraft().currentScreen).inventorySlots)
+			(((ContainerChest) ((GuiChest) Minecraft.getMinecraft().currentScreen).inventorySlots)
 				.getLowerChestInventory()
-				.getDisplayName()
-				.getUnformattedText()
-				.endsWith("Upgrades")))) {
+				.getStackInSlot(48)
+				!= null &&
+				((ContainerChest) ((GuiChest) Minecraft.getMinecraft().currentScreen).inventorySlots)
+					.getLowerChestInventory()
+					.getStackInSlot(48)
+					.getTooltip(Minecraft.getMinecraft().thePlayer, false).size() == 2 &&
```

Comment:
me omw to write a mod that adds an extra tooltip line to every item (on second thought: does this work with /neupackdev, since that adds extra lines?).
also having a variable for
 
			(((ContainerChest) ((GuiChest) Minecraft.getMinecraft().currentScreen).inventorySlots)
				.getLowerChestInventory()
				.getStackInSlot(48)
				
might be nice

Reply:
just dont have neupackdev enabled while doing it